notebooks/sycamore-tutorial-intermediate-etl.ipynb

```diff
@@ -111,7 +111,7 @@
    "source": [
     "We can visualize the pages of the PDF using the `show_pages` function, so you can see how APS chunked each page into elements.\n",
     "\n",
-    "Note that this requires the `poppler` library to be installed on your system. You can install it on a Mac with `brew install poppler`, or in Debian with `apt-get install poppler-utils`."
+    "Note that this requires the `poppler` library to be installed on your system. You can install it on macOS with `brew install poppler`, or in Linux with `apt-get install poppler-utils`."
```

Comment:
Debian != Linux. Now, people using other package manages than apt can probably figure it out